omc3/knob_extractor.py

```diff
@@ -279,16 +281,23 @@ def _extract(ldb, knobs_dict: KnobsDict, knob_categories: Sequence[str], time: d
 
             timestamps, values = knobvalue[knobkey]
             if len(values) == 0:
-                LOGGER.warning(f"No value for {knob} found")
+                LOGGER.debug(f"No value for {knob} found")
+                knobs_nan_or_inf.append(knob)
                 continue
 
             value = values[-1]
             if not math.isfinite(value):
-                LOGGER.warning(f"Value for {knob} is not a number or infinite")
+                LOGGER.debug(f"Value for {knob} is not a number or infinite")
+                knobs_nan_or_inf.append(knob)
                 continue
 
             LOGGER.info(f"Knob value for {knob} extracted: {value} (unscaled)")
             knobs[knob].value = value
+
+    if len(knobs_nan_or_inf):
+        LOGGER.info(f"The following knobs didn't return a value (or NaN/Inf):")
+        LOGGER.info(f" {', '.join(knobs_nan_or_inf)}")
```

Comment:
This does not work well with our logging setup as we append text at the right of this.

Reply:
you mean the `:`?
then just delete it? I thought it would be nice to split it up for readability or even log each missing knob in it's own log line

Reply:
I mean it will end up looking like this, iirc:
```
   INFO | The following knobs didn't return a value (or NaN/Inf): | .knob_extractor
   INFO | knob1, knob2, knob3, etc | .knob_extractor
```

Reply:
that's how it would look like, yes